src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -21,21 +20,17 @@ public class MemoryPoolTests
 
         public MemoryPoolTests()
         {
-            this.network = KnownNetworks.RegTest;
+            this.network = new BitcoinRegTest();
         }
 
         [Fact]
         public void AddToMempool()
         {
             using (NodeBuilder builder = NodeBuilder.Create(this))
             {
-                CoreNode stratisNodeSync = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet();
-
-                builder.StartAll();
+                CoreNode stratisNodeSync = builder.CreateStratisPowNode(this.network).NotInIBD().WithWallet().Start();
 
                 TestHelper.MineBlocks(stratisNodeSync, 105); // coinbase maturity = 100
```

Comment:
good call. Maybe we should call MineBlocks MineAndSync? Syncing is a bit of a hidden behavior.

Reply:
OK yeah we can rename in the next PR?

Reply:
sure